libs/core/src/lib/shared/interfaces/popover-position.ts

```diff
@@ -82,15 +89,32 @@ export const PopoverFlippedYDirection: {[key: string]: YPositions} = {
     'center': 'center'
 };
 
+export const PopoverFlippedRtlPlacement: {[key in RtlPlacement]: RtlPlacement} = {
+    'left-start': 'right-start',
+    'left': 'right',
+    'left-end': 'right-end',
+    'right-start': 'left-start',
+    'right': 'left',
+    'right-end': 'left-end',
+}
+
 export class PopoverPosition {
 
-    static getCdkPlacement(placement: Placement): ConnectedPosition {
+    static getCdkPlacement(placement: Placement, direction?: 'rtl' | 'ltr'): ConnectedPosition {
         const resultCdkPlacement = popoverPlacementMap[placement];
 
         if (!resultCdkPlacement && isDevMode()) {
             throw new Error('Invalid function argument. Check if "placement" is type of Placement Union');
         }
 
+        if (direction === 'rtl') {
+            const startPlaceStr = placement.split('-')[0];
+
+            if (startPlaceStr === 'left' || startPlaceStr === 'right') {
+                return popoverPlacementMap[PopoverFlippedRtlPlacement[placement]];
+            }
+        }
+
```

Comment:
You could've used replace method to flip the left/right :)
Then you wouldn't need the PopoverFlippedRtlPlacement const. But I accept this solution, looks good